install/docker-extension/ui/src/components/Catalog/Chart.js

```diff
@@ -1,8 +1,12 @@
 import BBChart from "../Chart/BBChart";
-import { CHART_COLORS, topicsList } from "../utils/constants";
+import { CHART_COLORS} from "../utils/constants";
```

Comment:
Did you fix the low contrast, dark axis displayed on the dark charcoal background?